app/assets/javascripts/infinite-scroll.js.coffee

```diff
@@ -0,0 +1,41 @@
+$ ->
```

Comment:
I can't personally vouch for any of these but it looks like there are some good tools to convert:
[The try CoffeeScript tab of the CoffeScript website](http://coffeescript.org/)
[js2coffee](http://js2.coffee/)
[decaffeinate](https://github.com/decaffeinate/decaffeinate)